web_src/less/markup/asciicast.less

```diff
@@ -1,5 +1,3 @@
-@import "../asciinema-player/dist/bundle/asciinema-player.css";
```

Comment:
BTW I'm surprised that this would have worked. CSS import from `node_modules` should be done like `asciinema-player/dist/bundle/asciinema-player.css`, not relative path.

Reply:
Actually, I tried `@import "asciinema-player/dist/bundle/asciinema-player.css";`, it doesn't work:

```text
ERROR in ./web_src/less/index.less (./web_src/less/index.less.webpack[javascript/auto]!=!./node_modules/css-loader/dist/cjs.js??clonedRuleSet-4.use[1]!./node_modules/less-loader/dist/cjs.js??clonedRuleSet-4.use[2]!./web_src/less/index.less)
Module build failed (from ./node_modules/css-loader/dist/cjs.js):
Error: Can't resolve 'markup/asciinema-player/dist/bundle/asciinema-player.css' in '/user_home/gitea/web_src/less'
```
This is all magic to me.

Reply:
Odd, it does certainly work for other imports from node_modules:

https://github.com/go-gitea/gitea/blob/main/web_src/less/_tribute.less#L1